requirements-dev.txt

```diff
@@ -1,2 +1,2 @@
 behave==1.2.5
-Markdown==2.5.2  # FIXME: 2.6.8 breaks tests
+Markdown==2.6.8
```

Comment:
I forget why this change was breaking tests earlier, but you may want to address everything else before upgrading this.